Rmd/stats_report.Rmd

```diff
@@ -118,11 +118,7 @@ This should be reviewed carefully to ensure that samples are correctly identifie
 
 ```{r metadata-report, collapse=TRUE, eval=TRUE}
 # table of sample metadata
-metadata_subset <- subset_metadata(designList[[params$display_group_filter]],
-                                   design_to_use, contrasts,
-                                   params$display_group_facet, params$display_group_filter)
-DESeqDesign_subset <- as.data.frame(metadata_subset$DESeqDesign)
-contrasts_subset <- as.data.frame(metadata_subset$contrasts)
+
```

Comment:
I'm not sure I follow why this would be deleted - isn't this necessary for the report to get the right data?